src/goLiveErrors.ts

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+import vscode = require('vscode');
+import { byteOffsetAt, getBinPath } from './util';
+import cp = require('child_process');
+import path = require('path');
+import { promptForMissingTool } from './goInstallTools';
+import { errorDiagnosticCollection } from './goMain';
+
+// Interface for settings configuration for adding and removing tags
+interface GoLiveErrorsConfig {
+	delay: number;
+	enabled: boolean;
+}
+
+let runner;
+
+export function goLiveErrorsEnabled() {
+	let goConfig = <GoLiveErrorsConfig>vscode.workspace.getConfiguration('go')['liveErrors'];
+	if (goConfig === null || goConfig === undefined || !goConfig.enabled) {
+		return false;
+	}
+	let autoSave = vscode.workspace.getConfiguration('files')['autoSave'];
+	if (autoSave !== null && autoSave !== undefined && autoSave !== 'off') {
```

Comment:
I know it is closed but to be honest, I don't see a reason why autosave is checked at all. I use autosave onFocusChange and why would it be clashing with a linter running after some delay? Even autosave afterDelay makes little sense to check as it may just be some much higher value than linter delay.